src/services/context_menu/context_menu_service.hpp

```diff
@@ -72,6 +72,18 @@ namespace big
 			         else
 				         g_notification_service->push_warning("Toxic", "Failed to take control of vehicle.");
 		         }},
+		        {"FIX VEHICLE",
+		           [this] {
+			            if (entity::take_control_of(m_handle))
+		        	    {
+			        	    VEHICLE::SET_VEHICLE_ENGINE_HEALTH(m_handle, 1000.f);
+				            VEHICLE::SET_VEHICLE_FIXED(m_handle);
+				            VEHICLE::SET_VEHICLE_DEFORMATION_FIXED(m_handle);
+				            VEHICLE::SET_VEHICLE_DIRT_LEVEL(m_handle, 0.f);
+			            }
+			            else
+				            g_notification_service->push_warning("Toxic", "Failed to take control of vehicle.");
```

Comment:
I don't think this falls under toxic does it?

Reply:
Could it be just "Warning" instead of "Toxic" notice?
Also Id rearrange list to remove "explode" from atop to not accidentaly blowup something like it happen so often.

Reply:
Yes, Warning would be the right fit for it. What would you suggest be at the top? iirc it saves the last option you were on

Reply:
Yes it saves, but for current game session only. So I did rearrange. Explode entity is strange behavior tbh. I moved it to bottom of list. Also I replace identity with outfit because it is more useful than frustrating if click on it and can't be revert until rejoin.